app/builders/immediate_email_builder.rb

```diff
@@ -15,56 +15,51 @@ def call
 
 private
 
-  attr_reader :subscription_content_changes
+  attr_reader :params
 
   def records
-    subscription_content_changes.map do |subscription_content_change|
-      single_email_record(subscription_content_change)
+    params.map do |param|
+      [
+        param[:address],
```

Comment:
I would use `.fetch` on these lines to ensure the key exists.

Reply:
Fixed in c7acb34